### PR TITLE
🔨 Added gtf capability to VEP

### PR DESCRIPTION
**Pull request by migbro**
:hammer: added ability to use gtf instead of cache for VEP
:book: updated PR template

# Pull Request Template

## Description

Added ability yo annotate using gtf instead of cache to VEP.  Added fastq splitter instruction for COV-IRT pipeline group.

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

- [x] This change requires a documentation update

## How Has This Been Tested?

Run successfully in private Cavatica project: https://cavatica.sbgenomics.com/u/d3b-bixu/rs-vpf5jbc3-cov-irt-controlled-access-study/tasks/#q

**Test Configuration**:
* Environment:
* Test files:

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -36,4 +36,5 @@ Please describe the tests that you ran to verify your changes. Provide instructi
 - [ ] I have added tests that prove my fix is effective or that my feature works
 - [ ] New and existing unit tests pass locally with my changes
 - [ ] Any dependent changes have been merged and published in downstream modules
-- [ ] I have checked my code and corrected any misspellings
\ No newline at end of file
+- [ ] I have checked my code and corrected any misspellings
+- [ ] Have I ensured that commits used for testing have been pushed before PR approval?
```
Comment (dmiller15) [inline .github/PULL_REQUEST_TEMPLATE.md:40]: very cool

**Review by dmiller15 — COMMENTED** [review]
It looks good but for the test examples can you show me these specific cases:
1. running with cache
1. running with merged cache
1. running with gtf

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (migbro) [reply .github/PULL_REQUEST_TEMPLATE.md:40]: Tests have been run here as requested: 
https://cavatica.sbgenomics.com/u/d3b-bixu/dev-rnaseq-snv/tasks/#q?search=vep-1oo-annotate&page=1

**Review by dmiller15 — APPROVED** [review]

**Comment by migbro** [comment]
Add info from https://cavatica.sbgenomics.com/u/danmiller/kf-gatk-cnv-dev/tasks/cb474dd3-574e-4e15-9ad6-2934efa3e7e9/, just to document results between KFDRC port and original tool are the same.,

**Review by mark-welsh — APPROVED** [review]

**Review by sickler-alex — APPROVED** [review]

**Pull request merged**